src/react/App.jsx

```diff
@@ -22,13 +22,63 @@ import AlertsProvider from './AlertManager';
 import Router from './Router';
 import Snackbar from './ui/Snackbar';
 
+function parseDashboardConfig(json) {
```

Comment:
We could move all the parsing closer to the initial config fetch and provide the React app with an already parsed config. I think that would improve 1) separation of concerns and 2) later refactoring with the parsing already happening in `index.js`

Reply:
About separation of concerns, The config is a concern of the App, maybe we should have a specific component just for the component, but if it is that small, i'd rather keep is as a function.
Also, Elm parses the config differently compared to react, so it would make sense to specialize the 2 code paths.
I prefer keeping index.js the bare minimum to what's required by Elm. If it's a react concern, i'd move the code inside a the react path (with the future benefit of typescript)